backend/src/main/java/com/devdive/backend/post/adapter/out/persistence/PostJpaEntity.java

```diff
@@ -0,0 +1,41 @@
+package com.devdive.backend.post.adapter.out.persistence;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity(name = "post")
+@Getter
+@Setter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class PostJpaEntity {
+
+    @Id
+    @Column(name = "post_id")
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @Column
+    private Long studyId;
+
+    @Column
+    private String thumbnail_url;
```

Comment:
이런 경우 컬럼의 네임 속성을 활용하는게 좋을 것 같습니다.
```suggestion
    @Column(name = "thumbname_url")
    private String thumbnailUrl;
```